modules/rostests/apitests/user32/keybd_event.c

```diff
@@ -0,0 +1,35 @@
+#include "precomp.h"
```

Comment:
Can you add a copyright header please? You can copy/adapt it from any other test. https://reactos.org/wiki/Coding_Style#File_Structure also has an example (and that's the latest version of the header, which is preferred).